language/German/strings.po

```diff
@@ -33,20 +33,26 @@ msgstr ""
 msgid ""
 "This system requires media images to be mounted for the following device(s): "
 msgstr ""
+"Für dieses System müssen Medienabbilder für die folgenden Geräte montiert "
```

Comment:
nobody in Germany says "montieren" for "to mount". A better translation would be "einhängen" or simply "laden"

Reply:
Fixed, don't know where that came from...